src/plist.h

```diff
@@ -320,6 +320,7 @@ EXPORT_INLINE Obj NewPlistFromArray(const Obj * list, Int length)
     Obj o = NEW_PLIST(T_PLIST, length);
     SET_LEN_PLIST(o, length);
     memcpy(BASE_PTR_PLIST(o), list, length * sizeof(Obj));
+    CHANGED_BAG(o);
```

Comment:
Aha, indeed, good catch! Let's hope this fixes our recent problems (I made `gac` use `NewPlistFromArgs`, which uses `NewPlistFromArray`, so it's plausible that this has now become a (more) noticeable problem.